library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOInputOperator.java

```diff
@@ -0,0 +1,582 @@
+/**
```

Comment:
Had added a test case. There are tests for all jdbc operators here library/src/test/java/com/datatorrent/lib/db/jdbc/JdbcOperatorTest.java